notebooks/4_1_software_framework.ipynb

```diff
@@ -62,36 +62,17 @@
    },
    "source": [
     "<div class=\"alert alert-box alert-info\">\n",
-    "To check that the WSL 2 instance, where the <em>Riallto</em> tools are installed, is enabled on your system, run the following cell:\n",
+    "To check that the WSL 2 instance, where the <em>Riallto</em> tools are installed, is enabled on your system, run the following cell (Note: this will not work on a Linux installation of Riallto):\n",
```

Comment:
Maybe we can split this in a command for Windows and one for Linux?

Reply:
What would the command for Linux check? The Ubuntu version?

Reply:
I've added a command to check the Linux Kernel version to ensure it is bigger than 6.10